docker/Dockerfile.bird-mixit-gpu

```diff
@@ -16,3 +16,5 @@ RUN curl $MODEL_CHECKPOINT_URL --output /tmp/model.zip && \
 ADD scripts /app/scripts
 
 WORKDIR /app
+
+RUN git clone https://github.com/google-research/sound-separation sound_separation
```

Comment:
This is a good solution; I'd replace the line that says `ADD vendor/sound-separation /app` with this, and make it checkout to `/app`. I think it's also okay to just `git submodule init` and `git submodule update` to instantiate the resources here.